template/_package.json

```diff
@@ -15,6 +15,11 @@
     <%_ if (test !== 'none') { _%>
     "test": "<%= test %>",
     <%_ } _%>
+    <%_ if (test === 'ava') { _%>
```

Comment:
`test === 'ava'` can be before `test !== 'none'`, and make `test !== 'none'` be `else if`